src/javascript/app/base/header.js

```diff
@@ -298,7 +298,7 @@ const Header = (() => {
 
                 switch (string) {
                     case 'unsubmitted': {
-                        result = verification_length === 2 && identity.status === 'none' && document.status === 'none';
+                        result = verification_length === 2 && identity.status === 'none' && document.status === 'none' && income.status === 'none';
```

Comment:
this verification length look a little bit confusing. why it should be exact  2?

Reply:
Hi @yauheni-kryzhyk-deriv , the verification length is to check how many items that should be verified in the 'needs_verification' array. It should be 3 since 'income' will be included into the 'needs_verification' array. Will edit and push again 🙏